src/mon/NVMeofGwMap.h

```diff
@@ -102,29 +107,31 @@ class NVMeofGwMap
     const NvmeGroupKey& group_key);
 
 public:
+
   int blocklist_gw(
     const NvmeGwId &gw_id, const NvmeGroupKey& group_key,
     NvmeAnaGrpId ANA_groupid, epoch_t &epoch, bool failover);
 
-  void encode(ceph::buffer::list &bl) const {
-    using ceph::encode;
-    ENCODE_START(1, 1, bl);
-    encode(epoch, bl);// global map epoch
-
-    encode(created_gws, bl); //Encode created GWs
-    encode(fsm_timers, bl);
-    ENCODE_FINISH(bl);
-  }
-
-  void decode(ceph::buffer::list::const_iterator &bl) {
-    using ceph::decode;
-    DECODE_START(1, bl);
-    decode(epoch, bl);
-
-    decode(created_gws, bl);
-    decode(fsm_timers, bl);
-    DECODE_FINISH(bl);
-  }
+    void encode(ceph::buffer::list &bl, uint64_t features) const {
+      uint8_t version = 1;
+      ENCODE_START(version, 1, bl);
```

Comment:
The indentation on this section is wrong.  Please re-indent this and every other line in this commit to match my cleanup.